runelite-client/src/main/java/net/runelite/client/game/AgilityShortcut.java

```diff
@@ -137,6 +139,14 @@ public boolean matches(Client client, TileObject object)
 	BRIMHAVEN_DUNGEON_MEDIUM_PIPE(34, "Pipe Squeeze", null, new WorldPoint(2698, 9501, 0), ObjectID.KARAM_DUNGEON_PIPE),
 	KOUREND_CATACOMBS_NORTH_EAST_CRACK_NORTH(34, "Crack", new WorldPoint(1715, 10057, 0), ObjectID.ZEAH_CATA_CRACK),
 	KOUREND_CATACOMBS_NORTH_EAST_CRACK_SOUTH(34, "Crack", new WorldPoint(1705, 10077, 0), ObjectID.ZEAH_CATA_CRACK),
+	VARROCK_CASTLE_GARDEN_TRELLIS(35, "Trellis", new WorldPoint(3227, 3471, 0), ObjectID.GARDEN_TRELLIS_CONCAVE_SHORTCUT)
+		{
+			@Override
+			public boolean matches(Client client, TileObject object)
+			{
+				return Quest.GARDEN_OF_TRANQUILLITY.getState(client) == QuestState.FINISHED;
+			}
+		},
```

Comment:
Dedent these one level. The outermost braces should be indented the same amount as the enum lines.